doc/nrf/known_issues.rst

```diff
@@ -750,7 +751,7 @@ KRKNWK-7831: Factory reset broken on coordinator with Zigbee shell
 
   **Workaround:** Call the ``bdb_reset_via_local_action`` function twice to remove all the network information.
 
-.. rst-class:: v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
```

Comment:
@maciejpietras , input needed for this one, as it is missing from https://projecttools.nordicsemi.no/confluence/display/KRKNWK/Release+notes+and+known+issues+2.0.0

Reply:
Valid also for NCS2.0.0